src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -7,6 +7,9 @@
   <PropertyGroup Condition=" '$(TargetsUnix)' == 'true' ">
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsLinux)' == 'true'">
+    <DefineConstants>TargetsLinux</DefineConstants>
+  </PropertyGroup>
```

Comment:
Can we do this a different way? e.g. expose the version from `PlatformDetection` rather than having to use ifdefs like this here?  It would return 0 on Windows, just as it does on Unix if we can't get the version.